### PR TITLE
Make CI work with the software rendering

### PR DESCRIPTION
**Pull request by Janix4000**

### CODE REVIEW

**Comment by Janix4000** [comment]
After a lengthy discussion, we created a separate, local CI job instead of a new orb. It won't be used anywhere else; in the future, it can be extracted if needed.

File: .circleci/config.yml [hunk 1]
```diff
@@ -34,12 +34,32 @@ jobs:
...
   build:
     jobs:
       - elixir/build_test
-      - elixir/test
+      - elixir_test_with_software_rendering
       - elixir/lint
       - rust_lint
       - rust_tests
+
+# VS Code Extension Version: 1.5.1
```
Comment (jerzywilczek) [inline .circleci/config.yml:65]: maybe remove this line

File: mix.lock [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   "bunch_native": {:hex, :bunch_native, "0.5.0", "8ac1536789a597599c10b652e0b526d8833348c19e4739a0759a2bedfd924e63", [:mix], [{:bundlex, "~> 1.0", [hex: :bundlex, repo: "hexpm", optional: false]}], "hexpm", "24190c760e32b23b36edeb2dc4852515c7c5b3b8675b1a864e0715bdd1c8f80d"},
-  "bundlex": {:hex, :bundlex, "1.0.1", "fd9e68b3636b8b5b678050f1f340e833e6d3890e110766398f38ec9c41da0b44", [:mix], [{:bunch, "~> 1.0", [hex: :bunch, repo: "hexpm", optional: false]}, {:qex, "~> 0.5", [hex: :qex, repo: "hexpm", optional: false]}, {:secure_random, "~> 0.5", [hex: :secure_random, repo: "hexpm", optional: false]}], "hexpm", "c6acfd675a4780c9b66c32c0e3c37b96b5314369857c5b7d3ffa198fa1e0085e"},
+  "bundlex": {:hex, :bundlex, "1.1.0", "16fff7cfd8a61394c4dc2b22fe6c478ba125f6e5798e3e81ef0bc07ff237bde4", [:mix], [{:bunch, "~> 1.0", [hex: :bunch, repo: "hexpm", optional: false]}, {:qex, "~> 0.5", [hex: :qex, repo: "hexpm", optional: false]}, {:secure_random, "~> 0.5", [hex: :secure_random, repo: "hexpm", optional: false]}], "hexpm", "9da48a102a924baf8b6161c7c1767b5d68eeba18dd9d37a241b8b2ddf87d8a8f"},
```
Comment (jerzywilczek) [inline mix.lock:4]: remove this file from the PR

**Review by jerzywilczek — CHANGES_REQUESTED** [review]
:cowboy_hat_face:

File: mix.lock (hunk 2 above)
Comment (Janix4000) [reply mix.lock:4]: No.
It is generated by `mix deps.get` (for some reason).

**Review by jerzywilczek — APPROVED** [review]
:rocket:

**Pull request merged**